src/app/src/components/Post.tsx

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import Button, { ButtonProps } from "@mui/material/Button";
+import { styled as Styled } from "@mui/material/styles";
+import styled from "styled-components";
+import logo from "../logo.svg";
+
+interface postItem {
+  Name: String;
```

Comment:
Suggest changing every "String" type to "string" type

See: [https://stackoverflow.com/questions/14727044/what-is-the-difference-between-types-string-and-string](url)

Reply:
Changed